src/com/oltpbenchmark/benchmarks/wikipedia/WikipediaLoader.java

```diff
@@ -181,7 +181,10 @@ private void loadUsers() throws SQLException {
         if(this.getDatabaseType() == DatabaseType.ORACLE) {
             // Oracle handles quoted object identifiers differently, do not escape names
             sql = SQLUtil.getInsertSQL(catalog_tbl, false);
+        } else if(this.getDatabaseType() == DatabaseType.CUBRID) {
```

Comment:
Please add comments to explain what this does.

Reply:
It means for the generated the SQL queries, include the column names and skip the first `auto_increment` column.
The field is set to be `auto_increment` since in the insert queries in the UpdatePage, these fields are not set and they have not null constraints. 
Both the loading phase and the procedures are inserting records. In the previous code, in the loading phase, it sets values for all columns. And in the procedures, it does not set value for the `auto_increment` fields. For Cubrid, it does not update the initial value of the `auto_increment` fields, while MySQL does this. So if we manually set values for these fields in the loading phase, in the execution phase Cubrid will set these values starting from some initial value which may already exist because of the loading phase. This causes the unique constraint violation. To avoid this, we skip setting these fields in the loading phase.